homeassistant/components/shelly/switch.py

```diff
@@ -167,14 +179,53 @@ def icon(self) -> str:
 
     async def async_turn_on(self, **kwargs: Any) -> None:
         """Open valve."""
+        async_create_issue(
+            self.hass,
+            DOMAIN,
+            "deprecated_valve_switch",
+            breaks_in_ha_version="2024.7.0",
+            is_fixable=False,
```

Comment:
Issues for deprecated services should be fixable so that the user can mark them as fixed when they have updated their automations and scripts.

Search for issues that have `is_fixable=True` for examples.